### PR TITLE
Added Support For Portuguese Portugal

### PR DESCRIPTION
**Pull request by psimoesSsimoes**
Tested with my own tests and against this website: 
http://www.languagesandnumbers.com/como-contar-em-portugues-europeu/pt/por-prt/

An important distinction has to be made for portuguese from portugal and from brasil.
It's explained in http://www.learningportuguese.co.uk/guide/reference/numbers

Finally, README is missing portuguese flag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +198,9 @@ fmt.Println(ntw.IntegerToDutch(42))
 fmt.Println(ntw.IntegerToTurkish(42))
 // Outputs: kırk iki
 
+fmt.Println(ntw.IntegerToPortuguesePT(42))
+// Outputs: kırk iki
```
Comment (moul) [inline README.md]: invalid output ?

File: cmd/ntw-web/main.go [hunk 2]
```diff
@@ -60,6 +59,8 @@ func server(c *cli.Context) error {
 			output = ntw.IntegerToSpanish(number)
 		case "tr", "turkish":
 			output = ntw.IntegerToTurkish(number)
+		case "pt-pt", "portuguesePT":
+			output = ntw.IntegerToTurkish(number)
```
Comment (moul) [inline cmd/ntw-web/main.go]: should be `ntw.InterToPortuguesePT` I thing

File: README.md (hunk 1 above)
Comment (psimoesSsimoes) [reply README.md]: no, i just forgot to alter it. It outputs: quarenta e dois

File: cmd/ntw-web/main.go (hunk 2 above)
Comment (psimoesSsimoes) [reply cmd/ntw-web/main.go]: yes, my bad

Comment (psimoesSsimoes) [reply cmd/ntw-web/main.go]: IntegerToPortuguesePT

**Comment by psimoesSsimoes** [comment]
travis is complaining because it tries to run ntw.IntegerToPortuguese_PT when it should be written ntw.IntegerToPortuguesePT. Sorry, I should have noticed this

**Comment by psimoesSsimoes** [comment]
i don't know what to do change to coverage results...

**Pull request merged**